src/taco-cli/test/help.ts

```diff
@@ -59,14 +58,14 @@ describe("help for a command", function (): void {
     }
 
     before(() => {
-        previous = process.env["TACO_UNIT_TEST"];
         process.env["TACO_UNIT_TEST"] = true;
     });
 
     after(() => {
         // We just need to reset the stdout just once, after all the tests have finished
         process.stdout.write = stdoutWrite;
-        process.env["TACO_UNIT_TEST"] = previous;
+        // Reset mocked out resources
+        process.env["TACO_UNIT_TEST"] = false;
     });
```

Comment:
because now we are trying to run all the test cases in the same process. same as how we will run coverage..